packages/eos-components/src/assets/EndlessLogo.svg

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<!-- Generator: Adobe Illustrator 19.2.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
```

Comment:
Is this the same SVG asset that we removed in 9c075ba0? Please explain in the commit message that this file is added, and where is it coming from.

Reply:
I got the Endless logo svg from here:
https://github.com/endlessm/gnome-initial-setup/blob/master/gnome-initial-setup/pages/language/EndlessLogo.svg

And modified the file to remove the text.

Reply:
OK, next time let's ask assets to Joana.